src/clj_kondo/impl/analyzer.clj

```diff
@@ -3339,27 +3364,9 @@
                             (types/add-arg-type-from-call ctx maybe-call expr)
                             (types/add-arg-type-from-expr ctx expr))
                           ret)))
-                    (cond
-                      (utils/boolean-token? function)
-                      (do (reg-not-a-function! ctx expr "boolean")
-                          (analyze-children (update ctx :callstack conj [nil t])
-                                            (rest children)))
-                      (utils/string-from-token function)
-                      (do (reg-not-a-function! ctx expr "string")
-                          (analyze-children (update ctx :callstack conj [nil t])
-                                            (rest children)))
-                      (utils/char-token? function)
-                      (do (reg-not-a-function! ctx expr "character")
-                          (analyze-children (update ctx :callstack conj [nil t])
-                                            (rest children)))
-                      (utils/number-token? function)
-                      (do (reg-not-a-function! ctx expr "number")
-                          (analyze-children (update ctx :callstack conj [nil t])
-                                            (rest children)))
-                      :else
-                      (do
-                        (types/add-arg-type-from-expr (update ctx :callstack conj [nil t]) expr)
-                        (analyze-children ctx children)))))
+                    (when (false? (analyze-non-symbol-function ctx children t function))
```

Comment:
Why is this when false? here? Doesn't this prevent linting children? Can't tell without reading the code in my editor, the layout in Github is wrapped.